workers/unity/Packages/com.improbable.gdk.mobile/iOS/Utility/DeviceInfo.cs

```diff
@@ -0,0 +1,12 @@
+using UnityEngine;
+
+namespace Improbable.Gdk.Mobile.Ios
+{
+    public class DeviceInfo
+    {
+        public static bool IsIosSimulator()
+        {
+            return SystemInfo.deviceModel.Equals("x86_64");
```

Comment:
is this enough to reliably detect an ios emulator?

Reply:
Yes, I think this is taken from UIDevice model value, which is always x86_84 for simulators. All actual devices are named "iPhone.." etc.

Reply:
@DanailPenev are we sure that this won't overlap with the android emulator values by accident?

Reply:
just want to check whether we need additional checks somewhere to clearly identify which x86_64 it is

Reply:
I mean it is code that should only ever be run on iOS devices.

Reply:
Oh, sorry, I misunderstood the question, was only thinking that we shouldn't mix it with iPhone device.
We can add `#if UNITY_IOS` to be extra safe

Reply:
never assume anything :P there will be people trying to do it

Reply:
though I guess that this code will be in a package that is only available in editor and ios,right?

Reply:
Thinking about it, someone could have `isiOSSimulator` and `isAndroidSimulator` in the same method, so I think it would be better to be safer to have a directive to compile this out

Reply:
The code won't compile for Android because the iOS assembly is only available for iOS and Editor.

Reply:
yeah, so it should be fine

Reply:
Fair enough, let's keep it as is in that case

Reply:
That's only fine if it doesn't compile with a comprehensible error message. Otherwise someone will spend ages banging their head against the wall of why them accessing the wrong identically named object doesn't compile.

Reply:
Good point, let me rename those to AndroidDeviceInfo and iOSDeviceInfo to address that